ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-periodics.yaml

```diff
@@ -106,22 +106,11 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: aws
     ci-operator.openshift.io/cloud-cluster-profile: aws-interop-qe
-    ci-operator.openshift.io/variant: opp-ocp4.15-interop
+    ci-operator.openshift.io/variant: opp-ocp4.16-interop
     ci.openshift.io/generator: prowgen
-    job-release: "4.15"
+    job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws
-  reporter_config:
-    slack:
-      channel: '#opp-discussion'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
-        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> {{end}}'
```

Comment:
Is slack integration removal on purpose?

Reply:
I added it back as it looks like it was removed after I ran the "make update" command